samples/app-dev/API-Development/REST/allOf_oneOf-anyOf_construct_schema/readme.md

```diff
@@ -0,0 +1,125 @@
+# allOf-oneOf-anyOf schema construct
+
+# Description
+
+In JSON Schema, the keywords allOf,anyOf,oneOf are used to specify that a data structure must validate against one or more specified schemas.
+The oneOf keyword specifies that the data must validate against exactly one of the schemas listed. If it matches more than one, or none at all, the validation fails.
+The anyOf keyword allows for validation against multiple schemas. If the data validates against at least one of the schemas listed in anyOf, it is considered valid.
+The allOf keyword JSON Schema is used to specify that a JSON document must satisfy all of the specified subschemas. It’s particularly useful for combining multiple schemas into one.
+This sample demonstrates some of the combinator keywords features present in the FLOGO ReceiveHTTPMessage trigger and InvokeRestService activity. Features which are covered in these sample apps are:
+
+## ReceiveHTTPMessage trigger
+1. Path, query and header parameters in the REST trigger.
+2. Configure multiple response code in REST trigger.
+3. Response Headers in REST trigger.
+4. App level schema in Request Schema and mapper activities.
+5. Multiple branching for each Response code.
+6. ConfigureHTTPResponse activity to map corresponding response schema and header with Return activity.
+
+## InvokeRestService activity
+1. Configuring InvokeRest activity with the API Spec of the producer REST service.
+2. Path, query, header parameters and Request and Response schema will be auto-populated.
```

Comment:
Pls remove Response schema will be auto-populated. and add allof, oneof and anyof keywords schema